src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.enums.Color;
+
+public abstract class Figure {
+    private final Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public abstract double square();
+
+    public abstract void draw();
```

Comment:
you should have interfaces to describe behaviour
AreaCalculator, Drawable

Reply:
I don't understand why this should be done when every (any) figure has these methods